Sources/ResilientDecoding/ErrorReporting.swift

```diff
@@ -142,3 +185,32 @@ private extension Error {
 }
 
 #endif
+
+// MARK: - Private
+
+/**
+ In the unlikely event that `enableResilientDecodingErrorReporting()` is called multiple times, this error will be reported to the earlier `ResilientDecodingErrorReporter` to signify that the later one may have eaten some of its errors.
+ */
+private struct MayBeMissingReportedErrors: Error { }
+
+/**
+ An error which is surfaced at the property level but is not reported via `ResilientDecodingErrorReporter` by default (it can still be accessed by providing the `includeUnknownNovelValueErrors: true` argument to `flushReportedErrors`). This error is meant to indicate that the client detected a type it does not understand but believes to be valid, for instance a novel `case` of a `String`-backed `enum`.
```

Comment:
> An error which is surfaced at the property level but is not reported via `ResilientDecodingErrorReporter` by default (it can still be accessed by providing the `includeUnknownNovelValueErrors: true` argument to `flushReportedErrors`).

This needs a revision based on your changes.